### PR TITLE
Add plugin: Rss Copyist

### PR DESCRIPTION
**Pull request by aoout**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/aoout/obsidian-rss-copyist

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
["description": "Get the rss articles as mdfiles.",](https://github.com/aoout/obsidian-rss-copyist/blob/24cdec13aa4a584d14ccb7b92adb5ce7faf7b8b0/manifest.json#L4)
Please properly capitalize RSS, and Markdown.

[newlest articels](https://github.com/aoout/obsidian-rss-copyist/blob/24cdec13aa4a584d14ccb7b92adb5ce7faf7b8b0/src/main.ts#L12), [newlest articels](https://github.com/aoout/obsidian-rss-copyist/blob/24cdec13aa4a584d14ccb7b92adb5ce7faf7b8b0/src/main.ts#L29), [articels](https://github.com/aoout/obsidian-rss-copyist/blob/24cdec13aa4a584d14ccb7b92adb5ce7faf7b8b0/src/main.ts#L40)
Typos.

**Comment by aoout** [comment]
thanks for the review.
I have fixed the typo, and updated the description.

**Comment by liamcain** [comment]
All of your commands and the readme say "[newlest](https://github.com/aoout/obsidian-rss-copyist/blob/dfff35690c712e25c608a1494cdc4b2d57a86387/src/main.ts#L12)". Is this supposed to be "newest"?

File: community-plugins.json [hunk 1]
```diff
@@ -9840,5 +9840,12 @@
     "author": "bandantonio",
     "description": "Import your Apple Books highlights and notes to Obsidian.",
     "repo": "bandantonio/obsidian-apple-books-highlights-plugin"
-  }
+  },
+  {
+      "id": "rss-copyist",
+      "name": "Rss Copyist",
```
Comment (liamcain) [inline community-plugins.json:9846]:
```suggestion
      "name": "RSS Copyist",
```

**Pull request merged**